src/pages/shipping.js

```diff
@@ -103,7 +105,7 @@ export const startShipping = async (userName) => {
         <div class="row">
     <div class="col-5">
 
-    <h3 style="text-align:center; margin-bottom:1rem;">Available Collections</h3>
+    <h4 style="text-align:center; margin-bottom:1rem;">Available Collections</h4>
```

Comment:
I checked the changes to the header sizes. I like the h4 elements better with the h5 elements.